src/main/java/com/example/libraryms/controllers/Dbconfig.java

```diff
@@ -0,0 +1,26 @@
+package com.example.libraryms.controllers;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class Dbconfig {
+    private static String DB_URL = "jdbc:mysql://localhost:3306/librarymanagementsystem";
+    private static String DB_USER = "root";
+    private static String DB_PASSWORD = "";
```

Comment:
don't pass sensitive information directly in our codebase. Use envirionment variables to hold sensitive information and reference the variable names.

fix: create a `.env` file in root directory

```
env

DB_PASSWORD=password
```

ref
```
private static String DB_PASSWORD=DB_PASSWORD
```